### ISSUE TITLE
Cannot find nc_set_var_chunk_cache_ints & nc_def_var_chunking_ints

### ISSUE THREAD

**Issue by afernandezody** [Unidata/netcdf-c#2096]
Hello,
I have upgraded netcdf-c to 4.8.1 from 4.8.0. However, I cannot find nc_set_var_chunk_cache_ints & nc_def_var_chunking_ints:
```
$ nm /opt/netcdf/lib/libnetcdf.so | grep nc_set_var_chunk_cache_ints
$ nm /opt/netcdf/lib/libnetcdf.a | grep nc_set_var_chunk_cache_ints
$ nm /opt/netcdf/lib/libnetcdf.so | grep nc_def_var_chunking_ints
$ nm /opt/netcdf/lib/libnetcdf.a | grep nc_def_var_chunking_ints
```
They all come back empty.
Thanks.

**Comment by DennisHeimbigner** [comment]
Odd. They are defined in netcdf-c/libhdf5/hdf5var.c.
See if you can find hdf5var.o and do an nm on that.

**Comment by afernandezody** [comment]
That is there but I built netcdf-c without HDF5 support (--disable-hdf5).

**Comment by DennisHeimbigner** [comment]
That explains it. I will add definitions for that case.

**Comment by edwardhartnett** [comment]
Do these functions have any meaning without hdf5?

**Comment by DennisHeimbigner** [comment]
We have seen this issue multiple times. When a function is meaningless in some
context (no HDF5 in this context), should we make provision and either ignore
the request or return an error. In this case, my inclination is to just ignore the
function, but other choices are defensible.

**Comment by edwardhartnett** [comment]
Yes, I get it. And we do have the NC_ENOTBUILT error for this exact purpose.

I guess I should learn from the nc_set_log_level() case, in which the sometimes missing function caused lots of projects (like PIO) to include their own version of the function to prevent link errors without having to use preprocessor statements to properly shield any use of the function.

So probably it does make sense to always include such functions, as we are forced to do in the dispatch table. 

I was just wondering @afernandezody what is the context in which this is a problem for you? What are you using netCDF for?

**Comment by afernandezody** [comment]
@edwardhartnett, 
I installed I/O API3.2 for CMAQ about 3 weeks ago with no problems in regards to netcdf components. I have now started a new system configuration (CMAQ was updated but it still needs the same I/O API version), but the compilation of I/O API with the newer netcdf beeped about nc_set_var_chunk_cache_ints & nc_def_var_chunking_ints. I guess that I could go back to v4.8.0 as it's probably going to make little to no difference performance-wise.
